src/main/kotlin/org/jetbrains/kotlin/jupyter/repl.kt

```diff
@@ -167,7 +171,7 @@ class ReplForJupyter(val scriptClasspath: List<File> = emptyList(),
 
     init {
         // TODO: to be removed after investigation of https://github.com/kotlin/kotlin-jupyter/issues/24
-        eval("1")
+        doEval("1")
```

Comment:
Is it just an optimization, or `eval()` will no longer work here?

Reply:
`eval()` uses library descriptors, that are now computed asynchronously, so we don't want to await them here. They will be awaited on first real execute request.